docs/sections/cn_workflows.md

```diff
@@ -211,7 +211,26 @@ popper run -f /home/user/proj/wf.yml -w /home/user/proj/
 
 The above writes the `/home/user/proj/myfile` even though Popper is 
 being invoked from `/tmp`, since the `-w` flag is being passed to 
-`ppopper run`.
+`popper run`.
+
+### Changing the working directory
+
+To specify a working directory for a step you can use the `dir` attribute
+in the workflow. This will only going to change where the specified
+command is executed.
+
+For example, adding `dir` to a workflow would looks like the following:
+
+```yaml
+version: '1'
+steps:
+- uses: docker://alpine:3.9
+  args: [touch, ./myfile]
+  dir: /path/to/dir/
+```
+
+It is worth mentioning that if the directory specified is outside the workspace,
```

Comment:
we can make it more explicit: "is specified outside the `/workspace` folder, then..."